assessment/asr_validation/wald2017b/analysis/A1-triaxial.i

```diff
@@ -150,7 +150,7 @@
   []
 []
 
-[Modules/TensorMechanics/LineElementMaster]
+[Physics/SolidMechanics/LineElementMaster]
```

Comment:
Aren't these supposed to be `Physics/SolidMechanics/LineElement/Quasistatic`?

Reply:
done